{"name": "user tries to log in with different failing credentials -- @1.3 ", "status": "passed", "steps": [{"name": "Given I go to start page", "status": "passed", "start": 1609114844382, "stop": 1609114846652}, {"name": "When I click log in button", "status": "passed", "start": 1609114846656, "stop": 1609114848647}, {"name": "When I put my email \"FakeUser3\" and password \"<password>\"", "status": "passed", "start": 1609114848651, "stop": 1609114848747}, {"name": "And I confirm the log in", "status": "passed", "start": 1609114848747, "stop": 1609114850992}, {"name": "Then wrong email or password shows up", "status": "passed", "start": 1609114850992, "stop": 1609114851028}, {"name": "And close browser", "status": "passed", "start": 1609114851028, "stop": 1609114852298}], "parameters": [{"name": "username", "value": "FakeUser3"}, {"name": "passwor", "value": "FakePasswor"}], "start": 1609114836353, "stop": 1609114852298, "uuid": "d4e8d8d3-7baa-4fe4-9885-da800877dc1a", "historyId": "2d19731a5e31dc05c589fda95d0b077e", "fullName": "features/login.feature:user tries to log in with different failing credentials", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "User logs"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}